docs/PowerAuth-Server-1.8.0.md

```diff
@@ -16,3 +16,11 @@ For manual changes use SQL scripts:
 
 Existing database index `pa_operation_status_exp` on the `pa_operation` table was modified to improve performance of the
 process of expiration of pending operations.
+
+### Add transports Column to pa_fido2_authenticator
```

Comment:
Inconsistent capitalization for `Add transports Column`.

Reply:
```suggestion
### Add Transports Column to pa_fido2_authenticator
```